config/dataset/mimic-iv/3.1/outputevents.yml

```
@@ -38,7 +38,7 @@ fields:
   - name: itemid
     type: int64
   - name: value
-    type: float32
+    type: int64
```

```suggestion
    type: float32
```

This column contains float values